tests/sweep_framework/sweeps/data_movement/embedding/embedding_pytorch2.py

```diff
@@ -0,0 +1,211 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional, Tuple
+
+import torch
+import random
+import ttnn
+
+from tests.ttnn.utils_for_testing import check_with_pcc, start_measuring_time, stop_measuring_time
+from models.utility_functions import torch_random
+
+TIMEOUT = 10
+# seed for random
+random.seed(0)
+
+
+def extract_brackets_content(line):
+    # Function to extract the content inside brackets
+    brackets_content = []
+    open_brackets = 0
+    current_content = ""
+
+    for char in line:
+        if char == "[":
+            open_brackets += 1
+            if open_brackets > 0:
+                current_content = ""  # Reset content inside the brackets
+        elif char == "]":
+            if open_brackets > 0:
+                brackets_content.append(current_content.strip())
+            open_brackets -= 1
+        elif open_brackets > 0:
+            current_content += char
+
+    return brackets_content
+
+
+def parse_md_file_simple_no_regex(file_path):
```

Comment:
Is this used in embedding at all?

Reply:
I think this is actually not, maybe we can take it out of this trace